pyanaconda/anaconda_logging.py

```diff
@@ -235,19 +239,35 @@ def forwardToJournal(self, logr, log_formatter=None, log_filter=None):
             journal_handler.setFormatter(log_formatter)
         logr.addHandler(journal_handler)
 
-    def stderrToJournal(self):
-        """Print all stderr messages from Anaconda to journal instead.
+    def redirect_glib_logging_to_journal(self):
+        """Redirect GLib based library logging to Journal.
+
+        Some GLib based libraries (such as GTK) do direct their
+        sometimes quite verbose log messages to the output of the
+        proces in which they are running. In the Anaconda case,
+        this creates issues with TTY1 being spammed with those
+        messages, with important content (such RDP connection intructions)
+        being scrolled out of view.
 
-        Redirect Anaconda main process stderr to Journal, as otherwise this could end up writing
-        all over the TUI on TTY1.
+        :param log: anaconda log handler
         """
+
         if not self.write_to_journal:
             return
 
-        # create an appropriately named Journal writing stream
-        anaconda_stderr_stream = journal.stream("anaconda", priority=journal.LOG_ERR)
-        # redirect stderr of this process to the stream
-        os.dup2(anaconda_stderr_stream.fileno(), sys.stderr.fileno())
+        # create functions that convert the messages comming
+        # from GLib into something that fits to the PYthon logging format
```

Comment:
s/comming/ s/PYthon/python/

It's not actually a python logging format but the anaconda logging format right?